src/posts/CMA-DMA-Nov-22.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'News from UK and EU'
+date: '2022-11-28'
+tags: ['Policy', 'Updates']
+---
+
+Howdy, [Bruce](https://brucelawson.co.uk) here, with an update on progress we've made in both the EU and with the UK's competition regulator, the Competition and Markets Authority (CMA).
+
+## Browser engines on the agenda
+
+We've had encouraging success at putting browser engines on the regulators' agenda. While rendering engines seem pretty esoteric, it's of paramount importance, as Apple has done a great job in concealing their anti-competitive behaviour, saying "you can have other browsers on iOS", while glossing over the fact that all other browsers are Apple's preferred version of WebKit under the hood. Many web developers don't realise this, which is why we made this meme:
+
+{% image "src/images/blog/apple-browser-ban.webp", "Fred from Scooby Doo with a masked figure and text 'Firefox on iPhone'. Fred removes the mask to reveal the villain headed 'Apple's Safari'. Then the same with Edge on iPhone and Chrome on iPhone." %}
```

Comment:
I think the build is failing because of the addition of _src/_ here in the path
```sh
`Template render error` was thrown
11:26:14 AM: Failed during stage 'building site': Build script returned non-zero exit code: 2 (https://ntl.fyi/exit-code-2)
11:26:14 AM: [11ty] > ENOENT: no such file or directory, stat 'src/src/images/blog/apple-browser-ban.webp'
11:26:14 AM: `Template render error` was thrown:
11:26:14 AM: [11ty]     Error: ENOENT: no such file or directory, stat 'src/src/images/blog/apple-browser-ban.webp'
```

Reply:
Ah yes the `{% image %}` shortcode already prepends that to the string, so you end up with `src/src/...` here.